stations.csv

```diff
@@ -27048,15 +27048,15 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29194;Wien Franz-Josefs-Bahnhof;wien-franz-josefs-bahnhof;;;48.226165;16.361266;26862;AT;Europe/Vienna;f;f;f;f;t;f;;;f;;f;8100446;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29195;Pamhagen;pamhagen;;;47.704099;16.903432;;AT;Europe/Vienna;f;f;f;t;f;f;;;f;;f;8100448;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29196;Wien Penzing;wien-penzing;;;48.192707;16.304436;26862;AT;Europe/Vienna;f;f;f;t;t;f;;;f;;f;8100450;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;Vienna Penzing;Viena Penzing;Vienne Penzing;Vienna Penzing;;Wenen;;;;;;Wiedeń Penzing;;;;;
-29197;Bergheim bei Salzburg;bergheim-bei-salzburg;;;47.83935;13.020964;;AT;Europe/Vienna;f;f;f;t;f;f;;;f;;f;8100456;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29197;Bergheim bei Salzburg;bergheim-bei-salzburg;;;47.83935;13.020964;6994;AT;Europe/Vienna;f;f;f;t;f;f;;;f;;f;8100456;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
I think only station in the city Salzburg should be added as child of Salzburg

Reply:
Ok I'll modify it, but it's in the suburbs so I wasn't sure.